pkg/scheduler/schedule_one.go

```diff
@@ -129,6 +129,11 @@ func (sched *Scheduler) ScheduleOne(ctx context.Context) {
 			sched.handleBindingCycleError(bindingCycleCtx, state, fwk, assumedPodInfo, start, scheduleResult, status)
 			return
 		}
+		// Update the PodCondition condition to reflect that the pod is scheduled.
+		podutil.UpdatePodCondition(&pod.Status, &v1.PodCondition{
```

Comment:
Can you add some checks for this in tests to ensure the condition is updated?
For example, we can add some check to see if pod condition is updated after scheduling in `TestSchedulerScheduleOne`.

Reply:
Okay, I went back and checked the issue.
Me myself feel reasonable to let scheduler set the `PodScheduled` Condition. But this is not mentioned in the issue. The issue simply wants this condition to be owned.
In this case, I think we'll need to wait for sig-node folks to see if it's ok with them.

BTW, can we let the condition be owned by the kubelet, but still allow scheduler to set it to false?(I'm not familiar with this part). Even if we can do this, it feels a little weird.

Reply:
Seems we both feel logical to let the scheduler own the condition.
Then can you add some tests for this?
I think we'll need a UT to make sure condition is updated after ScheduleOne.(Maybe just add a check in existing tests like I said before).
Any maybe an e2e test to check the `managedFields` after pod scheduled?